app/core/resources/constants/settings.py

```diff
@@ -5,12 +5,12 @@
 
 from app.core.resources.config_loader import read_config
 
-NUMBER_OF_WORKERS = int(read_config(section="libreoffice", value="workers"))
+NUMBER_OF_WORKERS: int = int(read_config(section="libreoffice", value="workers"))
 
-LIBRE_OFFICE_PATH = read_config(section="libreoffice", value="path")
+LIBRE_OFFICE_PATH: str = read_config(section="libreoffice", value="path")
 
 # LOG
-log_section = "log"
-LOG_FORMAT = read_config(section=log_section, value="format", raw=True)
-LOG_PATH = read_config(section=log_section, value="path")
-LOG_LEVEL = str(read_config(section=log_section, value="level")).upper()
+log_section: str = "log"
```

Comment:
Is there a reason for the lowecase?

Reply:
I usually do section in lower case and config variable in upper case so that it is easier to read. I should change log_section to _log_section to mark it as private

Reply:
I approve of making it private because it is more clear that it is only used there!